db/migrate/20210527193617_add_feed_format_to_traffic_cams.rb

```diff
@@ -0,0 +1,5 @@
+class AddFeedFormatToTrafficCams < ActiveRecord::Migration[5.2]
+  def change
+    add_column :traffic_cams, :feed_format, :string
```

Comment:
does `default` need to be added before `null: false` or can it be same time? A little confusing about how rails runs migrations, is there a way to see what SQL ends up running here?

Reply:
`null: false` requires a `default: <value>` or existing data. The latter can be accomplished with:
```
add_column :traffic_cams, :feed_format, :string
execute "UPATE traffic_cams SET feed_format = '...'"
change_column :traffic_cams, :feed_format, :string, null: false
```
If you want to use a modal to build the update statement, you create a temporary modal class inside the migration to prevent breaking the migration with future modal changes.

In this particular case, I think `null: false, default: 'image'` is sufficient.

Reply:
This is ready for re-review @cbarber @chewnoill